### PR TITLE
Maven central publishing

### LINKED ISSUE #195: publish to maven central instead of bintray
😢 

- https://github.com/sbt/sbt/issues/6294
- https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

### PR DESCRIPTION
**Pull request by retronym**
Vale bintray.

Fixes #195

### CODE REVIEW

**Comment by retronym** [comment]
Tested with a snapshot release:

```
sbt:root> publishSigned
[info] :: delivering :: pl.project13.scala#sbt-jmh;0.4.1-SNAPSHOT :: 0.4.1-SNAPSHOT :: integration :: Wed May 05 14:20:17 AEST 2021
[info] 	delivering ivy file to /Users/jz/code/sbt-jmh/plugin/target/scala-2.12/sbt-1.0/ivy-0.4.1-SNAPSHOT.xml
[info] :: delivering :: pl.project13.scala#sbt-jmh;0.4.1-SNAPSHOT :: 0.4.1-SNAPSHOT :: integration :: Wed May 05 14:20:18 AEST 2021
[info] 	delivering ivy file to /Users/jz/code/sbt-jmh/plugin/target/scala-2.12/sbt-1.0/ivy-0.4.1-SNAPSHOT.xml
[info] 	published sbt-jmh to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/sbt-jmh-0.4.1-SNAPSHOT.jar.asc
[info] 	published sbt-jmh to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/sbt-jmh-0.4.1-SNAPSHOT-sources.jar
[info] 	published sbt-jmh to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/sbt-jmh-0.4.1-SNAPSHOT.jar
[info] 	published sbt-jmh to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/sbt-jmh-0.4.1-SNAPSHOT-javadoc.jar
[info] 	published sbt-jmh to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/sbt-jmh-0.4.1-SNAPSHOT-javadoc.jar.asc
[info] 	published sbt-jmh to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/sbt-jmh-0.4.1-SNAPSHOT-sources.jar.asc
[info] 	published ivy to https://oss.sonatype.org/content/repositories/snapshots/pl/project13/scala/sbt-jmh_2.12_1.0/0.4.1-SNAPSHOT/ivy-0.4.1-SNAPSHOT.xml
```

**Comment by retronym** [comment]
@dwijnand Does this look right?

File: RELEASING.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ Releasing notes
 - dry run:
   - `sbt -Dsbt.supershell=false clean ^plugin/publishLocalSigned )`
 - release the plugin:
+  -  prerequisite: @ktoso needs to grant your oss.sontaype.org publish permissions by means of a request to [OSSRH-1324](https://issues.sonatype.org/browse/OSSRH-1324).
```
Comment (johanandren) [inline RELEASING.md:9]: Alternatively we could perhaps just change the groupId of the plugin to one we have publish permissions for already?

Comment (retronym) [reply RELEASING.md:9]: You and I both have permissions for the current group ID. What alternative did you have in mind?

Comment (johanandren) [reply RELEASING.md:9]: Ah, we do, I had forgotten that and thought this was a reminder for us. Everything good then!

**Review by johanandren — APPROVED** [review]

**Review by dwijnand — APPROVED** [review]

**Pull request merged**